tests/periph_wdt/main.c

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (C) 2019 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test for wdt Drivers
+ *
+ *              This test initializes the wdt counter to a preset value
+ *              and spins until a wdt reset is triggered
+ *
+ * @author      Francisco Molina <francois-xavier.molina@inria.fr>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "periph/wdt.h"
+#include "periph_cpu.h"
```

Comment:
I think this include can be removed, since it's might be already included by `periph/wdt.h`.